Offseason2022/src/main/java/frc/robot/subsystems/Vision.java

```diff
@@ -83,7 +83,15 @@ public void periodic( )
       m_targetVertAngle = m_yfilter.calculate(m_table.getEntry("ty").getDouble(0.0));
       m_targetArea = m_table.getEntry("ta").getDouble(0.0);
       m_targetSkew = m_table.getEntry("ts").getDouble(0.0);
-      m_targetValid = m_table.getEntry("tv").getBoolean(false);
+      if (m_table.getEntry("tv").getDouble(0.0) > 0)

```

Comment:
Let's do this on a single line. We are assigning the variable m_targetValid to be either true or false based on a condition. So

m_targetValid = (m_table.getEntry("tv").getDouble(0.0) > 0);

The parentheses around the comparison will evaluate to "true" or "false" and then be assigned to the variable. Just like if that comparison was used for an "if" statement.